music_assistant/providers/digitally_incorporated/__init__.py

```diff
@@ -567,17 +661,33 @@ def _channel_to_radio(self, channel_data: dict[str, Any], network_key: str) -> R
 
         prov_id = f"{network_key}:{channel_key}"
         channel_name = str(channel_data.get("name", "Unknown"))
-        network_info = NETWORKS[network_key]
 
-        # Create metadata with optional image
+        # Create metadata with optional image and genres
         metadata = MediaItemMetadata(
-            description=f"{network_info['description']} - {channel_name}",
+            description=self._get_description(channel_data),
             explicit=False,
         )
 
+        # Add genre information
+        try:
+            genres = await self._get_channel_genres(channel_data, network_key)
```

Comment:
so, is this avoidable to do for EVERY station ?

Reply:
We should be able to grab it once for each site in the network, and use the returned information to determine the genre info for all stations in that site. It ships be reasonable for us to do this during provider scans and just cache the results.

Reply:
Yeah, that will be better. Otherwise you are going to a extra call for every station in a listing.